src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSCloud.java

```diff
@@ -276,6 +295,17 @@ public String getDisplayName() {
             return Messages.DisplayName();
         }
 
+        public ListBoxModel doFillAwsRegionItems() {
```

Comment:
Yeh, I know, but there no such option for ECS. Even more, ec2 availability do not match to ECS availability.